index.html

```diff
@@ -13,26 +13,64 @@
   <h2>Tic Tac Toe</h2>
   <div class="board">
     <div id="row-1">
-      <div id="cell-0"></div>
-      <div id="cell-1"></div>
-      <div id="cell-2"></div>
+      <div class = box id="cell-0"></div>
+      <div class = box id="cell-1"></div>
+      <div class = box id="cell-2"></div>
     </div>
     <div id="row-2">
-      <div id="cell-3"></div>
-      <div id="cell-4"></div>
-      <div id="cell-5"></div>
+      <div class = box id="cell-3"></div>
+      <div class = box id="cell-4"></div>
+      <div class = box id="cell-5"></div>
     </div>
     <div id="row-3">
-      <div id="cell-6"></div>
-      <div id="cell-7"></div>
-      <div id="cell-8"></div>
+      <div class = box id="cell-6"></div>
+      <div class = box id="cell-7"></div>
+      <div class = box id="cell-8"></div>
     </div>
   </div>
 
-  <div class="status">
-    <button id="start" type="button">Start Game</button>
+  
+
+
+    <button id="start" type="button" >Start Game</button>
+  
+  
+
+  <div id="turn">
+
   </div>
+
+
+
+
   <script src="main.js"></script>
+
+<div id="display"></div>
+  <script>
+    let display = document.getElementById('display')
+    let count = 0
+
+
+    time()
+    function time() {
+      setInterval(countUp, 1000)
+    }
```

Comment:
You don't really need the `time` function here. This would work the same if you just did the `setInterval` in the global scope. Nice encapsulation though